dist/scss/wds-mixins/_community-header-theming.scss

```diff
@@ -0,0 +1,135 @@
+@import '../wds-functions/index';
+
+@mixin community-header-theming(
+	$header-background-color,
+	$color-text,
+	$color-links,
+	$color-page,
+	$color-page-border
+) {
+
+	// the same logic as $is-dark-wiki in oasis
+	$is-dark-theme: lightness($color-page) < 50;
+
+	$text-color: $wds-color-white;
+	@if luminance($header-background-color) > 0.3 {
+		$text-color: $wds-color-black;
+	}
+	$text-hover-color: rgba($text-color, 0.8);
+	$text-hover-transition: color .2s;
```

Comment:
transition duration value is common for text, backgrounds, borders and fills - it should be put in a separate variable.